DailyDemo/View Controllers/CallViewController.swift

```diff
@@ -109,10 +109,11 @@ class CallViewController: UIViewController {
         self.roomURLField.text = self.roomURLString
 
         // Update inputs to enable video local view prior to joining:
-        self.inputs = try! self.callClient.updateInputs(
-            .set(
-                .init(camera: .set(.isEnabled(true)))
-            ))
+        self.inputs = try! self.callClient.updateInputs { inputs in
```

Comment:
Note: this will conflict with change [here](https://github.com/daily-co/daily-x/pull/705/files).